code/API_definitions/number-verification.yaml

```diff
@@ -277,7 +284,18 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorInfo'
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 403
+                  code:
+                    enum:
+                      - PERMISSION_DENIED
+                      - NUMBER_VERIFICATION.USER_NOT_AUTHENTICATED_BY_MOBILE_NETWORK
+                      - INVALID_TOKEN_CONTEXT
```

Comment:
@bigludo7 this one is no longer needed I think. 
Per current design, the token must 3-legged identifying a subscription and if the phone number in the request doesn't match we return a false in the response.

We are removing this one in almost all apis

Reply:
@fernandopradocabrillo Just to be sure before we remove it - it means that test case @NumberVerification_phone_number_verify204_no_phonenumber_associated_with_access_token is not relevant anymore?